pymemcache/client/base.py

```diff
@@ -739,6 +739,10 @@ def _raise_errors(self, line, name):
             error = line[line.find(b' ') + 1:]
             raise MemcacheServerError(error)
 
+    # extract the key helper to support extensions
```

Comment:
Because we're doing this to support possible extension via subclasses, could you add a little bit more documentation here so that the contract is clearer to end users? I think using a docblock would probably be better than a comment for that purpose, too.